src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/__tests__/autocomplete.command.where.test.ts

```diff
@@ -150,7 +150,9 @@ describe('WHERE <expression>', () => {
         ...getFieldNamesByType('any')
           .map((field) => `${field} `)
           .map(attachTriggerCommand),
-        ...allEvalFns.filter((fn) => fn.label !== 'QSTR' && fn.label !== 'MATCH'),
+        ...allEvalFns.filter(
+          (fn) => fn.label !== 'QSTR' && fn.label !== 'KQL' && fn.label !== 'MATCH'
```

Comment:
nit: Can we use here FULL_TEXT_SEARCH_FUNCTIONS from the shared/constants.ts ?

Reply:
updated in e8a355e583284e2b0b14aaccd18474bbf42ffa59